python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -515,7 +574,7 @@ def _get_task_overrides(
 
         return overrides
 
-    def _get_run_task_kwargs_from_run(self, run: DagsterRun) -> Mapping[str, Any]:
+    def _get_run_task_kwargs_from_run(self, run: DagsterRun) -> Dict[str, Any]:
```

Comment:
can this stay `Mapping`?

Reply:
When I change this to Mapping I get a bunch of other errors from pyright because it doesn't define `__set_item__`, `__delitem__`, or `update` - honestly I'm not sure how this code was passing pyright before I updated to Dict. That's part of why I updated it, I was getting intellisense errors in PyCharm before updating to Dict.

Reply:
But I can still change it back, maybe the build uses some options in its pyright check that allows these errors?

Reply:
Sure no problem! I guess if we wanted to be pedantic about it, we would maybe want to guard against the case where `json.loads()` returns a list with a ValueError, since a more accurate type hint as it is right now might be Mapping | Sequence